setup.cfg

```
@@ -1,11 +1,10 @@
 [flake8]
 max-line-length = 120
 exclude = docs/src, build, dist, .ipynb_checkpoints
-extend-ignore = E741
```

ignore errors about test assertions `assert type(x) is type(y)`